tests/chainer_tests/test_variable.py

```diff
@@ -2824,13 +2824,14 @@ def setUp(self):
         self.x = np.random.uniform(-1, 1, self.in_shape).astype(self.dtype)
         self.y = np.random.uniform(-1, 1, self.in_shape).astype(self.dtype)
 
-    def check_loss_scale(self, x_data, y_data):
+    def check_loss_scale(self, x_data, y_data, chainerx=False):
```

Comment:
It shadows the global variable `chainerx` which have chainerx module, so it seems better to use another variable name.
```suggestion
    def check_loss_scale(self, x_data, y_data, use_chainerx=False):
```
It seems also OK to pass `xp` as an argument.
```suggestion
    def check_loss_scale(self, xp, x_data, y_data):
```